Module_Postman/activities/Act2_Creating_and_Testing_BitBloxs/8.md

```diff
@@ -0,0 +1,18 @@
+# POST Request
+
+Now that we are finished with GET request, let's take a look at POST request.
+
+While GET request is for retrieving information, for example, we retrieve the information about box with our GET request POST request is for inserting information.
+
+In this example, we will be coloring white boxes with our POST request.
+
+According to [the documentation]( https://documenter.getpostman.com/view/4691925/SW15zHJv?version=latest#a633bb6c-c481-497e-8696-72129056398e), the route for changing the color of a box is `https://bitbloxs.herokuapp.com/change/<int:box_id>/<string:color>`.
+
+So go ahead and replace `https://bitbloxs.herokuapp.com/` with our environment variable, `url`, replace `<int:box_id>` with the box ID that you want to modify, and replace `<string:color>` with the color you want to change your box into.
+
+![ ](https://projectbit.s3-us-west-1.amazonaws.com/darlene/postman/Change+Color.PNG)
+
+Don't forget to set the Authorization setting! Otherwise, you are be good to go!
+
+Go ahead and send a request, and watch the box change color!
```

Comment:
Since things are getting a bit more complex am wondering about adding a troubleshooting tip if the request doesn't behave as expected here e.g. open the postman console and have a look at the request detail..?